tests/Middleware/RequestInterfaceMiddlewareTest.php

```diff
@@ -11,38 +11,36 @@
 
 namespace Sentry\Tests\Middleware;
 
-use PHPUnit\Framework\TestCase;
-use Psr\Http\Message\ServerRequestInterface;
 use Sentry\Configuration;
 use Sentry\Event;
 use Sentry\Middleware\RequestInterfaceMiddleware;
 use Zend\Diactoros\ServerRequest;
 use Zend\Diactoros\Uri;
 
-class RequestInterfaceMiddlewareTest extends TestCase
+class RequestInterfaceMiddlewareTest extends MiddlewareTestCase
 {
     public function testInvokeWithNoRequest()
     {
         $configuration = new Configuration();
         $event = new Event($configuration);
 
-        $invokationCount = 0;
-        $callback = function (Event $eventArg) use ($event, &$invokationCount) {
+        $invocationCount = 0;
+        $callback = function (Event $eventArg) use ($event, &$invocationCount) {
             $this->assertSame($event, $eventArg);
 
-            ++$invokationCount;
+            ++$invocationCount;
         };
 
         $middleware = new RequestInterfaceMiddleware();
         $middleware($event, $callback);
 
-        $this->assertEquals(1, $invokationCount);
+        $this->assertSame(1, $invocationCount);
```

Comment:
This test should use the new `assertMiddlewareInvokesNextCorrectly` method

Reply:
I'm not really convinced that we should have a default in the test (in this case creating the request and exception instances everytime), but I think that I understand you did it to avoid having to create mocks in each of the middleware tests. We could anyway refactor the code so that we assert the middleware was invoked using a `boolean` variable instead of a counter.

Reply:
I'm not really convinced that we should have a default in the test (in this case creating the request and exception instances everytime), but I think that I understand you did it to avoid having to create mocks in each of the middleware tests. We could anyway refactor the code so that we assert the middleware was invoked using a `boolean` variable instead of a counter.